scripts/initialize.sh

```diff
@@ -33,7 +36,11 @@ help()
    echo -e "\t--beneficiary: Staking beneficiary address"
    echo -e "\t--authorizer: Staking authorizer address"
    echo -e "\t--stake-amount: Staking amount"
-   echo -e "\t--authorization-amount: Authorization amount\n"
+   echo -e "\t--authorization-amount: Authorization amount"
+   echo -e "\t--tbtc-path: 'Local' tbtc project's path. A temporary folder with tbtc is created and removed"\
+                           "upon installation if the path is not provided"
+   echo -e "\t--skip-ecdsa-deployment: This option skips ecdsa and tbtc deployment. Default is false"
```

Comment:
Since we are talking about initialization, should we `--skip-ecdsa-initialization` rather than skipping its *deployment*?